src/lib/ekoEmbed/v2.js

```diff
@@ -78,11 +80,15 @@ class EkoEmbedV2 {
         this.eventEmitter.once(eventName, callback);
     }
 
+    dispose() {
+        window.removeEventListener('message', this.onEkoEventFired);
```

Comment:
Same comment here, recommend refactoring this into a `removeIframeListeners()` method.